### ISSUE TITLE
gsub_reverseccchain misdocumented as gsub_revesechain

### ISSUE THREAD

**Issue by sh1boot** [fontforge/fontforge#5471]
While I spelled 'reverse' correctly despite the documentation, I didn't realise I needed an extra `c` before `chain`, so this mistake really threw me.

https://github.com/fontforge/fontforge/blob/705433c295326df4a16e5491df917d0b48427242/doc/sphinx/scripting/python/fontforge.rst?plain=1#L4057

**Comment by iorsh** [comment]
Yes, this stands for "**reverse** **c**ontextual **chain**", thanks for noticing!
